cloudzero/README.md

```diff
@@ -0,0 +1,59 @@
+# Agent Check: CloudZero
+
+## Overview
+
+This check monitors [CloudZero][1].
```

Comment:
Can you please add some detail here on:
1. What CloudZero is/does
2. The type of data you'll be sending in/pulling out of Datadog
3. What value your integration will bring customers